index.html

```diff
@@ -1112,6 +1112,71 @@ <h2>Thing Description Server</h2>
                     </p>
                 </dd>
             </dl>
+            <dl>
+                <dt>CoAP</dt>
+                <dd>
+                <p>
+                    <span class="rfc2119-assertion" id="service-coap-secure">
+                        A CoAP-based TD Server providing a <a>TD</a> SHOULD use the appropriate transport
+                        security mechanism for the transport protocol used (i.e., DTLS for CoAP over UDP
+                        or TLS for CoAP over TCP).
+                    </span>
+                    <span class="rfc2119-assertion" id="service-coap-access-control">
+                        A CoAP-based TD Server providing a <a>TD</a>
+                        SHOULD provide the resource only after performing necessary
+                        authentication and authorization.
+                    </span>
+                </p>
+                <p>
+                    <span class="rfc2119-assertion" id="service-coap-method">
+                        A CoAP-based TD Server providing a <a>TD</a>
+                        MUST serve that resource with a `GET` method.
+                    </span>
+                    <span class="rfc2119-assertion" id="service-coap-resp">
+                        A successful response from a CoAP-based TD Server providing a <a>TD</a>
+                        MUST have a 2.05 (Content) status, contain a Content-Format option
+                        with value 432 (`application/td+json`), and the <a>TD</a> in the payload.
```

Comment:
we can leave this as is, but I'm wondering if the part here about the Content-Format (and the corresponding part under the HTTP section) conflicts with what we have done for Directories to make this more flexible.  HOWEVER we can go ahead and merge this and maybe just raise an issue to think about making these consistent.